app/src/main/java/com/vero/tasky/core/presentation/components/RootNavigation.kt

```diff
@@ -0,0 +1,29 @@
+package com.vero.tasky.core.presentation.components
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.vero.tasky.core.presentation.navigation.Screens
+
+@Composable
+fun RootNavigation(
+    navController: NavHostController,
+    isLoggedIn: Boolean
+) {
+    NavHost(
+        navController = navController,
+        startDestination = if (isLoggedIn) Screens.Agenda.route else Screens.Login.route,
+        modifier = Modifier.fillMaxSize()
```

Comment:
Is there a use to having this as a fillMaxSize tho?

Reply:
Yeah, I thought that without this NavHost wouldn't take all possible space